users/config.rst

```diff
@@ -980,6 +1173,75 @@ setLowPriority
     disable this behavior, for example to control process priority yourself
     as part of launching Syncthing, set this option to ``false``.
 
+.. option:: options.maxFolderConcurrency
+
+    This option controls how many folders may concurrently be in I/O-intensive
+    operations such as syncing or scanning.  The mechanism is described in
+    detail in a :doc:`separate chapter </advanced/option-max-concurrency>`.
+
+.. option:: options.crashReportingURL
+
+    Server URL where :doc:`automatic crash reports <crashrep>` will be sent if
+    enabled.
+
+.. option:: options.crashReportingEnabled
+
+    Switch to opt out from the :doc:`automatic crash reporting <crashrep>`
+    feature to avoid Syncthing "phoning home" on serious troubles.  Defaults to
```

Comment:
I'd suggest to make this one explicit, i.e. replace "phoning home" with "sending panic logs" (with a `ref` to https://docs.syncthing.net/users/crashrep.html).

Reply:
It already is a ref to that page.  "Phoning home" is an established term AFAICT, which should trigger anybody who's sensitive to such behavior in a piece of software.  That was exactly my goal.

Reply:
"Phoning home" is usually used with a negative connotation though. I don't think you'll find any other piece of software calling its own data or crash collection with this particular term.

Reply:
Yeah, there will be exceptions. Still, when people hear about "phoning home", it's usually closer to something like https://techsupportalert.com/how-to-stop-windows-10-from-phoning-home or https://news.softpedia.com/news/samsung-phones-said-to-come-with-chinese-spyware-phoning-home-528768.shtml, which implicitly indicates nefarious activities, so I'd still suggest to use a more "neutral" expression here.

Reply:
I agree with Tomasz here: We want people to give a choise, but we also want people to send crash reports. Just the neutral statement he proposed (to prevent Syncthing from sending crash reports) is more appropriate and even has more meaning (phoning home can mean basically anything).

Reply:
Done.